src/main/java/reader/DataReader.java

```diff
@@ -75,9 +74,9 @@ public void writeFile(List<String> json) {
       FileWriter myWriter = new FileWriter("localisation.json");
       myWriter.write(jsonString);
       myWriter.close();
```

Comment:
close() should be most probably called in finally block  so the writer will be closed even though exception was thrown
https://www.baeldung.com/java-finally-keyword